core-api/src/main/java/com/optimizely/ab/config/audience/Condition.java

```diff
@@ -23,5 +23,5 @@
  */
 public interface Condition {
 
-    boolean evaluate(Map<String, String> attributes);
+    boolean evaluate(Map<String, ?> attributes);
```

Comment:
Update the header on this file to be `2016-2018`. Please also do it for other files you've touched in this commit.